src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2657,8 +2657,13 @@ private function registerRateLimiterConfiguration(array $config, ContainerBuilde
         }
     }
 
+    /**
+     * @deprecated since Symfony 6.2 without any replacement
+     */
     public static function registerRateLimiter(ContainerBuilder $container, string $name, array $limiterConfig)
     {
+        trigger_deprecation('symfony/framework-bundle', '6.2', 'The "%s" method is deprecated without any replacement.', __METHOD__);
```

Comment:
that's trigger a self deprecation in registerRateLimiterConfiguration that would be fixed

Reply:
of course. A virtual `bool $triggerDeprecation` argument (retrieved via func_get_arg()) should do the trick.